### PR TITLE
Allow clustered integration tests to run easily in an IDE (close #2118)

### LINKED ISSUE #2118: Can't run Clustered integration tests in IntelliJ
Because of the shading on "dist" and "clustered-dist", some classes are missing from the client classpath.

We should find a way to add them one way or another.

### PR DESCRIPTION
**Pull request by henri-tremblay**

### CODE REVIEW

File: clustered/integration-test/build.gradle [hunk 1]
```diff
@@ -37,6 +37,7 @@ dependencies {
   testCompile "com.fasterxml.jackson.core:jackson-databind:2.8.0"
   testRuntime project(':clustered:clustered-dist')
   testRuntime project(':dist')
+  testRuntime "org.terracotta:runnel:$terracottaPlatformVersion"
```
Comment (ljacomet) [inline clustered/integration-test/build.gradle]: How come this is the only change needed and not around the scope of the `compileOnly` ones?

Comment (henri-tremblay) [reply clustered/integration-test/build.gradle]: I don't know. I just know that adding only this worked. Unless IntelliJ remembered a bit too much the state with all `compileOnly` removed. Try it for yourself.

Comment (ljacomet) [reply clustered/integration-test/build.gradle]: Ok did a test, it works if that is declared as `testCompileOnly` which allows it to not leak in the normal gradle build - so please change it.

**Comment by ljacomet** [comment]
When I use IntelliJ and `<ctrl>+<shift>+R` to run a test, the default working directory is the project root and not `<root>/clustered/integration-test` which makes all search of the kit and `gradlew` fail.

So while this is a good idea, it is not fully ready.

**Comment by henri-tremblay** [comment]
Héhé... This is a long standing story with IntelliJ. Everything starts from the root. Which is a problem because Maven (and Gradle from as far as I know) are starting from the module. To solve that, you normally should always set `$MODULE_DIR$` as the default directory for JUnit templates. This aligns them.

I can try to guess where I am but still think this is a sensible default :-)

**Comment by henri-tremblay** [comment]
Ok. I now support both starting directories.

File: clustered/integration-test/build.gradle (hunk 1 above)
Comment (henri-tremblay) [reply clustered/integration-test/build.gradle]: Now in `testCompileOnly`

**Comment by henri-tremblay** [comment]
test this please

**Pull request merged**